test/modelcheck/mps-prj/.mps/modules.xml

```diff
@@ -2,7 +2,7 @@
 <project version="4">
   <component name="MPSProject">
     <projectModules>
-      <modulePath path="$PROJECT_DIR$/solutions/my.solution/my.solution.msd" folder="" />
+      <modulePath path="$PROJECT_DIR$/$mypath$/my.solution/my.solution.msd" folder="" />
```

Comment:
Please don't do this. Using Macros in `modules.xml` is not supported in MPS 2019.1+.

Reply:
How about undoing this in the test project and just adding this to the readme example then (so that others won't need to search for it again, as I had to)?